integration/gethnetwork/main/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/obscuronet/obscuro-playground/integration/gethnetwork"
+)
+
+// Spins up a new Geth network.
+func main() {
+	config := ParseCLIArgs()
+
+	gethBinaryPath, err := gethnetwork.EnsureBinariesExist(gethnetwork.LatestVersion)
+	if err != nil {
+		panic(err)
+	}
+	gethnetwork.NewGethNetwork(config.StartPort, gethBinaryPath, config.NumNodes, 1, config.PrefundedAddrs)
+	fmt.Println("Geth network started.")
+
+	select {}
```

Comment:
Should we read for a CTRL+C / Interrupt and do a clean exit ?

Using this will leave zombie processes wont it ?

Reply:
Yeah. I've never done that - any good examples?

Reply:
```go
c := make(chan os.Signal, 1)
signal.Notify(c, os.Interrupt)
go func(){
    for sig := range c {
        // sig is a ^C, handle it
    }
}()

```

something like https://stackoverflow.com/questions/11268943/is-it-possible-to-capture-a-ctrlc-signal-sigint-and-run-a-cleanup-function-i

I think that works, maybe just capture other interrupt signals like `syscall.SIGTERM`